package.json

```diff
@@ -40,13 +44,14 @@
     "godaddy-js-style-eslint": "./bin/eslint"
   },
   "devDependencies": {
-    "eslint": "^7.25.0",
+    "eslint": "^8.4.1",
```

Comment:
Will we want a global `peerDependencies` bump for `eslint` to be on `^8` in case the plugin bumps are not compatible with older versions of `eslint`?

Reply:
Just so I understand, you are suggesting that we add eslint@8 as a peer dep at the root package.json?

Reply:
No, to individual packages (unless adding to root does some magic and adds to each package)